x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/analytics/action/AnalyticsStatsAction.java

```diff
@@ -33,6 +33,17 @@ private AnalyticsStatsAction() {
         super(NAME, Response::new);
     }
 
+    /**
+     * Items to track. Serialized by ordinals. Append only, don't remove or change order of items in this list.
```

Comment:
Should we have a test somewhere that enforces this?  E.g. enums that extend writeable directly can implement `AbstractWriteableEnumTestCase`.  Not the same situation here, but having something conceptually similar to `testValidOrdinals()` would be nice.  Just verifies that BOXPLOT == 0, etc etc

Reply:
Indeed we need a bwc test for usage. Right now it is completely broken so it didn't make sense to have it. But as soon as I back port this I will open a PR for bwc test for usage.